scripts/cterm-colors-generator.lua

```diff
@@ -0,0 +1,43 @@
+-- vim-colortemplate is needed
+-- the current working directory must be set to the repo root
+-- it's best to have the main file already open in another window
+
+-- IMPORTANT: the `cterm_color` key must always be below the `color` key of the
+--            same icon. Currently they are.
+
+-- :source this file when ready
+
+local fn = vim.fn
+
+vim.cmd("packadd vim-colortemplate")
```

Comment:
Works nicely, not sure we need this, as it will be automatically loaded from &rtp

Reply:
If you're sure it's not needed I'll remove it.

Reply:
As long as it's in `start` it's not needed.